pyQuARC/main.py

```diff
@@ -188,6 +188,8 @@ def validate(self):
 
     @staticmethod
     def _error_message(messages):
+        if messages is None:
```

Comment:
why is `messages` None in the first place? did we check if there was some other issue that's causing "messages" to be None? That should be the first thing you check instead of putting a bandaid here

Reply:
okay, thanks. I will look into that.